code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1314,6 +1314,19 @@ GLOBAL_LIST_EMPTY(custom_outfits) //Admin created outfits
 		if(ADMIN_PUNISHMENT_PIE)
 			var/obj/item/reagent_containers/food/snacks/pie/cream/nostun/creamy = new(get_turf(target))
 			creamy.splat(target)
+		if (ADMIN_PUNISHMENT_CUSTOM_PIE)
+			var/obj/item/reagent_containers/food/snacks/pie/cream/nostun/A = new(get_turf(target))
+			if(!A.reagents)
+				var/amount = input(usr, "Specify the reagent size of [A]", "Set Reagent Size", 50) as num
```

Comment:
Oh yea, pies have a reagent holder already, this bit is useless. What you may want is to make a subtype that starts off with no reagent inside.

Reply:
nah, I'd have to rework the initialize on reagent_containters, so nope, not doin that